### PR TITLE
Add comment indicator guide

### PR DESCRIPTION
**Pull request by josephbaylon**
#### Summary
- Added a new item for comment indicator guide.

#### Ticket Link
NA

### CODE REVIEW

File: site/content/contribute/mobile/e2e/guide-for-writing.md [hunk 1]
```diff
@@ -33,7 +33,10 @@ Before writing a script, ensure that it has a corresponding test case in Test Ma
     Example:
     - `send.button`
     - `post.<post-id>`
-3. Simulate user interaction using available [actions](https://github.com/wix/Detox/blob/master/docs/APIRef.ActionsOnElement.md).
-4. Verify user interface (UI) expectation using [expect](https://github.com/wix/Detox/blob/master/docs/APIRef.Expect.md).
-5. When using `action`, `match`, or another API specific to particular platform, verify that the equivalent logic is applied so that the API does not impact the other platform. Always run tests in both platforms.
-6. See Detox [documentation](https://github.com/wix/Detox/tree/master/docs) for reference.
+3. Prefix each comment line with appropriate indicator. Each line in a multi-line comment should be prefixed accordingly. Separate and group test step comments and assertion comments for better readability.
```
Comment (josephbaylon) [inline site/content/contribute/mobile/e2e/guide-for-writing.md:36]: This is the new item (`3`)

**Review by saturninoabril — APPROVED** [review]
Thanks @josephbaylon!

File: site/content/contribute/mobile/e2e/guide-for-writing.md (hunk 1 above)
Comment (enahum) [reply site/content/contribute/mobile/e2e/guide-for-writing.md:36]: definitely not did this in my PR :S

**Review by enahum — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]
This looks great!

**Pull request merged**